examples/dataflow-python-examples/requirements.txt

```diff
@@ -1,4 +1,4 @@
-apache-beam[gcp]==2.4.0
+apache-beam[gcp]==2.11.0
 Faker==0.8.13
```

Comment:
These `Faker` and `faker-schema` things are unused. remove.

Reply:
Done.